src/configure.ac

```diff
@@ -69,6 +69,7 @@ AC_ARG_ENABLE([memdbg], [AC_HELP_STRING([--enable-memdbg], [Enable memdbg memory
 AC_ARG_ENABLE([asan], [AS_HELP_STRING([--enable-asan], [* Build with AddressSanitizer])], [asan=$enableval], [asan=no])
 AC_ARG_ENABLE([ubsan], [AS_HELP_STRING([--enable-ubsan], [* Build with UndefinedBehaviorSanitizer])], [ubsan=$enableval], [ubsan=no])
 AC_ARG_ENABLE([ubsantrap], [AS_HELP_STRING([--enable-ubsantrap], [* Build with UndefinedBehaviorSanitizer, crashes on failure])], [ubsantrap=$enableval], [ubsantrap=no])
+AC_ARG_ENABLE([simd], [AS_HELP_STRING([--disable-simd], [* Build forcing SIMD logic to be OFF])], [simd=$enableval], [simd=yes])
```

Comment:
Super minor nitpick: I dislike saying "off" here and I also dislike over-use of capitals. IMHO this should say "Build forcing SIMD logic to be disabled".

Reply:
I will make this fix.